docs/managing-environments.md

```diff
@@ -76,6 +76,41 @@ special `system` Python version to retrieve the default behavior:
 poetry env use system
 ```
 
+## Activating the environment
+
+`poetry env activate` command prints the activate command to the console. This way you won't leave the current shell.
+You can then feed the output to `eval` to activate the environment. This way is the closest to manually activating the environment.
+
+{{% note %}}
+Looking for `poetry shell`? It was moved to a plugin: [`poetry-plugin-shell`](https://github.com/python-poetry/poetry-plugin-shell)
+{{% /note %}}
```

Comment:
I'd either put this at the beginning or the end and not between the text and the examples how to feed the output to the eval command of different shells.